apps/nextjs-website/src/lib/cmsApi.ts

```diff
@@ -136,6 +140,21 @@ export const getQuickStartGuidesProps = async () => {
   }
 };
 
+export const getUrlReplaceMapProps = async () => {
+  const {
+    config: { FETCH_FROM_STRAPI: fetchFromStrapi },
+  } = buildEnv;
+
+  if (fetchFromStrapi) {
+    const urlReplaceMap = await fetchUrlReplaceMap(buildEnv);
+    return Object.assign(
+      {},
+      staticUrlReplacesMap,
+      makeUrlReplaceMap(urlReplaceMap)
+    );
+  }
```

Comment:
I think that, for now, you should return staticUrlReplaceMap if `fetchFromStrapi`  is `false`